### PR TITLE
docs(developer): extra context help for keyboard-editor (does not exist in Keyman Developer 17.0)

### PR DESCRIPTION
**Pull request by Meng-Heng**
Fixes: #2131

@keymanapp-test-bot skip

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

File: developer/src/tike/xml/help/contexthelp.xml [hunk 1]
```diff
@@ -182,7 +182,20 @@
...
       <p>Tests your KeymanWeb keyboard in an Internet Explorer embedded window</p>
     </Control>
+    <Control Name="pagesTouchLayout" Title="Touch Layout page">
+      <p>Touch Layout tab defines the functionality of the keyboard for mobile devices:
+      <ul>
+        <li>Top bar Controls</li>
+        <li>Keyboard area</li>
+        <li>Middle bar controls</li>
+        <li>Long press area</li>
+      </ul>
```
Comment (mcdurdin) [inline developer/src/tike/xml/help/contexthelp.xml:192]: This is probably not helpful documentation. Is it copied from help.keyman.com?

Comment (Meng-Heng) [reply developer/src/tike/xml/help/contexthelp.xml:222]: I couldn't find it in KD 17.0 or help.keyman.com, so I followed a similar pattern of "pages" in ControlName and added the content as seen. It is possible that this section is from an older version, but I haven't found exactly where it is either.

Comment (mcdurdin) [reply developer/src/tike/xml/help/contexthelp.xml:222]: OK, I am not sure where to go with this. It doesn't seem worth writing more detailed documentation here, so perhaps we need to update the help site and then copy that over?

**Review by mcdurdin — APPROVED** [review]

**Comment by mcdurdin** [comment]
Going ahead and merging this. We can improve docs in a future PR.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.58-alpha](https://keyman.com/downloads/releases/alpha/18.0.58)

File: developer/src/tike/xml/help/contexthelp.xml (hunk 1 above)
Comment (Meng-Heng) [reply developer/src/tike/xml/help/contexthelp.xml:222]: I'm happy to work on that. I hope I can find it in version 15.0 or so, and then update the documents there. If users are still using older versions, they will see the context help.

**Pull request merged**